src/control/lib/daos/pool_property.go

```diff
@@ -340,6 +340,61 @@ func PoolProperties() PoolPropertyMap {
 				},
 			},
 		},
+		"checkpoint": {
+			Property: PoolProperty{
+				Number:      PoolPropertyCheckpointMode,
+				Description: "WAL Checkpointing behavior",
+			},
+			values: map[string]uint64{
+				"disabled": PoolCheckpointDisabled,
+				"timed":    PoolCheckpointTimed,
+				"lazy":     PoolCheckpointLazy,
+			},
+		},
+		"checkpoint_freq": {
+			Property: PoolProperty{
+				Number:      PoolPropertyCheckpointFreq,
+				Description: "WAL Checkpointing frequency",
+				valueHandler: func(s string) (*PoolPropertyValue, error) {
+					rbErr := errors.Errorf("invalid Checkpointing Frequency value %s", s)
+					rsPct, err := strconv.ParseUint(strings.ReplaceAll(s, "%", ""), 10, 64)
+					if err != nil {
+						return nil, rbErr
+					}
+					return &PoolPropertyValue{rsPct}, nil
```

Comment:
Looks like a little copypasta here... ;) This is just a simple integer, right? Should probably remove the stuff having to do with stripping '%', etc.